cogs/info.py

```diff
@@ -137,11 +137,23 @@ async def dialog(self, ctx: commands.Context, *, args: str = ''):
             await ctx.send('Sorry, but the message limit is 140 characters <:hiroJey:692008426842226708>')
             return
 
+        if len(re.findall('(<@!\d+>)', text)) > 0:
```

Comment:
text channel mentions :)

Reply:
and role mentions, already working on it 😉